docs/api/cases/cases-api-get-reporters.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[cases-api-get-reporters]]
+== Get reporters API
+++++
+<titleabbrev>Get reporters</titleabbrev>
+++++
+
+Returns all case reporters (users who opened cases).
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/reporters`
+
+`GET <kibana host>:<port>/s/api/cases/reporters`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the cases you're seeking.
+
+=== Query parameters
+
+`owner`::
+(Optional, string) A filter to limit the retrieved reporters to a specific set 
+of applications. If this parameter is omitted, the response will contain all 
+reporters that the user has access to read.
```

Comment:
I think the user has access to read cases (in one of three features), so it's clearer to add that qualifier:
```suggestion
reporters from cases that the user has access to read.
```
If you agree, we should likely make the same change in the "get tags" API too. i.e. "If not specified, the response contains all tags from cases that the user has access to read"